app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
```

Comment:
no need for the foreign_key here.

Reply:
Hey, @dasileker I had to make it because this command `first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post')` wasn't working I had to make it like `first_post = Post.create(user: first_user, title: 'Hello', text: 'This is my first post')` this to work.
So should I change it also?